src/deprecated.jl

```diff
@@ -1,8 +1,16 @@
 ###########################################
 # v0.20 deprecations:
 ##
-Base.@deprecate_binding DiscreteSurface CellGrid true
-Base.@deprecate_binding ContinuousSurface VertexGrid true
+
+function DiscreteSurface(args...; kwargs...)
+    Base.depwarn("Makie.DiscreteSurface() is deprecated, use Makie.CellGrid() instead", :DiscreteSurface)
```

Comment:
Should we actually make this a warning with `maxlog=1`?
Every time we remove deprecations in a breaking version, people are confused why suddenly things are broken without a warning, since depwarns are turned off almost everywhere it matters for a plotting package.

Reply:
yeah I think that's better

Reply:
Ok, changed to `@warn` in 5c3829cc2.